org-configurations/default-configuration.yml

```diff
@@ -4,7 +4,7 @@ slug: "default"
 
 # configuration variables for the server app
 server:
-  host: "http://127.0.0.1:8000"
+  host: "http://ec2-18-191-159-195.us-east-2.compute.amazonaws.com:8000"
```

Comment:
why are you changing this? It looks like something that should not be done to me

Reply:
Really sorry. I keep forgetting to change that back before committing. I've changed it back now though.